src/PureScript/Backend/Chez/Runtime.purs

```diff
@@ -0,0 +1,53 @@
+module PureScript.Backend.Chez.Runtime where
+
+import Prelude
+
+import Data.Array.NonEmpty as NEA
+import Data.Maybe (Maybe(..))
+import PureScript.Backend.Chez.Syntax (ChezDefinition(..), ChezExport(..), ChezImport(..), ChezImportSet(..), ChezLibrary)
+import PureScript.Backend.Chez.Syntax as S
+
+runtimeModule :: ChezLibrary
+runtimeModule =
+  { "#!chezscheme": true
+  , "#!r6rs": true
+  , name:
+      { identifiers: NEA.cons' "_Chez_Runtime" [ "lib" ]
+      , version: []
+      }
+  , exports:
+      [ ExportIdentifier "boolean>?"
+      , ExportIdentifier "boolean>=?"
+      , ExportIdentifier "boolean<=?"
+      , ExportIdentifier "boolean<?"
+      ]
+  , imports:
+      [ ImportSet $ ImportPrefix
+          (ImportLibrary { identifiers: NEA.singleton "chezscheme", version: Nothing })
+          "scm:"
+      ]
+  , body:
+      { definitions:
+          [ DefineUncurriedFunction "boolean->integer" [ "x" ] $ S.List
+              [ S.Identifier "scm:if"
+              , S.Identifier "x"
+              , S.Integer $ S.LiteralDigit "1"
+              , S.Integer $ S.LiteralDigit "0"
+              ]
+          , makeBooleanComparison ">?"
+          , makeBooleanComparison ">=?"
+          , makeBooleanComparison "<=?"
+          , makeBooleanComparison "<?"
```

Comment:
It would actually need to be something like this instead:
```
tell $ [ ExportIdentifier functionName ]
```

I think `StateT` is more apt here

Reply:
> It would actually need to be something like this instead:

Ah... right.

> I think StateT is more apt here

Why State? That allows you to modify things whereas Writer does not. Or are you anticipating doing more than just `tell` here in the future?

Reply:
I guess it's just less icky to do `modify_ $ Array.cons ...` than `tell $ [ ... ]` for me

Reply:
Well... Icky or not, I think my point still stands.